styles.css

```diff
@@ -132,15 +132,21 @@ body {
 }
 
 .mdl-mini-footer {
-  height: 120px;
+  height: 128px;
```

Comment:
Hey, check out the `vh` concept instead of `px`.
https://developer.mozilla.org/en-US/docs/Web/CSS/length